Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PowderDiffILLDetScanReduction.py

```diff
@@ -182,6 +185,8 @@ def PyExec(self):
         self._progress.report('Normalising to monitor')
         if self.getPropertyValue('NormaliseTo') == 'Monitor':
             input_group = NormaliseToMonitor(InputWorkspace=input_group, MonitorID=0)
+            if instrument.getName() == 'D2B':
+                input_group = Scale(InputWorkspace=input_group, Factor=1e+6)
```

Comment:
It is not ideal to have this just for D2B. Does Gabriel really need this? I think it was only done in LAMP because integers are used for the counts somewhere.

Reply:
This is needed for convenience, since eventually the data is exported to ascii with a limited number of decimal digits. So not to lose significant digits, data has to be scaled up by 1M.